src/constants/intervals.js

```diff
@@ -5,10 +5,11 @@ export const WEEK = 'WEEK'
 export const MONTH = 'MONTH'
 export const YEAR = 'YEAR'
 
-const INTERVALS = [
-    { value: DAY, label: i18n.t('Day') },
-    { value: WEEK, label: i18n.t('Week') },
-    { value: MONTH, label: i18n.t('Month') },
-    { value: YEAR, label: i18n.t('Year') },
-]
+const INTERVALS = {
+    [DAY]: { value: DAY, label: i18n.t('Day') },
+    [WEEK]: { value: WEEK, label: i18n.t('Week') },
+    [MONTH]: { value: MONTH, label: i18n.t('Month') },
+    [YEAR]: { value: YEAR, label: i18n.t('Year') },
+}
```

Comment:
This style is a lot nicer and accessing the `{ value, label }` directly and gives a balance between verbosity and complexity. :+1: